.github/workflows/deploy-production-apps.yml

```
@@ -82,6 +82,7 @@ jobs:
 
     - name: Download artifact
       uses: dawidd6/action-download-artifact@v2
+      continue-on-error: true
```

is this going to continue on error for all apps or just smart contract apps?

---

Yes, this rule on github ignores all errors. Smart contract apps doesn't use this workflow so it doesn't matter. Frontend apps use. If if it errors for a frontend app (which is very unlikely) it will still error on the next step for them

---

what about other kinds of apps, e.g. subgraphs?

---

Only frontend apps use this workflow. Subgraphs are deployed manually, there's no pipeline for deploying them atm

---

In other words, it's safe to trigger error here. `Download artifact` step is only for frontend apps and if something fails downloading the artifact, deploying the app to production will also fail (the next step on the workflow)

---

For future reference: it errors on that step if there is no change done on any frontend app, so we need to continue in those cases.